TheGodfather/TheGodfather.csproj

```diff
@@ -73,4 +73,12 @@
     </EmbeddedResource>
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
think these changes should be in pr #13  please let me know if this is accurate

Reply:
Probably, but does it matter?